symbology/render-multilayer-symbols/src/main/java/com/esri/samples/render_multilayer_symbols/RenderMultilayerSymbolsSample.java

```diff
@@ -0,0 +1,459 @@
+
+        // set the size of the layer and create a new multilayer point symbol from it
+        pictureMarkerSymbolLayer.setSize(40);
+        var multilayerPointSymbol = new MultilayerPointSymbol(List.of(pictureMarkerSymbolLayer));
+
+        // create location for the symbol
+        Point point = new Point(-80, 20 - offset, SpatialReferences.getWgs84());
+
+        // create graphic with the location and symbol and add it to the graphics overlay
+        var graphic = new Graphic(point, multilayerPointSymbol);
+        graphicsOverlay.getGraphics().add(graphic);
+      } else if (pictureMarkerSymbolLayer.getLoadStatus() == LoadStatus.FAILED_TO_LOAD) {
+        new Alert(Alert.AlertType.ERROR, "Picture marker symbol layer failed to load").show();
+      }
+    });
+    // load the picture marker symbol layer
+    pictureMarkerSymbolLayer.loadAsync();
+
+  }
+
+  /**
+   * Adds new graphics constructed from multilayer point symbols.
+   * 
+   * @param multilayerSymbol the multilayer symbol to construct the vector marker symbol element with
+   * @param geometry the input geometry for the vector marker symbol element
+   * @param offset the value used to keep a consistent distance between symbols in the same column
+   */
+  private void addGraphicsWithVectorMarkerSymbolElements(MultilayerSymbol multilayerSymbol, Geometry geometry, double offset) {
+    
+    // define a vector element and create a new multilayer point symbol from it
+    var vectorMarkerSymbolElement = new VectorMarkerSymbolElement(geometry, multilayerSymbol);
+    var vectorMarkerSymbolLayer = new VectorMarkerSymbolLayer(List.of(vectorMarkerSymbolElement));
+    var multilayerPointSymbol = new MultilayerPointSymbol(List.of(vectorMarkerSymbolLayer));
+    
+    // create point graphic using the symbol and add it to the graphics overlay
+    var graphic = new Graphic(new Point(-150, 20 - offset, SpatialReferences.getWgs84()), multilayerPointSymbol);
+    graphicsOverlay.getGraphics().add(graphic);
+    
+  }
+
+  /**
+   * Adds new graphics constructed from multilayer polyline symbols.
+   *
+   * @param dashSpacing The pattern of dots/dashes used by the line.
+   * @param offset The value used to keep a consistent distance between symbols in the same column.
+   */
+  private void addLineGraphicsWithMarkerSymbols(List<Double> dashSpacing, double offset) {
+
+    // create a dash effect from the provided values
+    var dashGeometricEffect = new DashGeometricEffect(dashSpacing);
+
+    // stroke used by line symbols
+    var solidStrokeSymbolLayer = new SolidStrokeSymbolLayer(3.0, ColorUtil.colorToArgb(Color.RED), List.of(dashGeometricEffect));
+    solidStrokeSymbolLayer.setCapStyle(StrokeSymbolLayer.CapStyle.ROUND);
+
+    // create a polyline for the multilayer polyline symbol
+    var polylineBuilder = new PolylineBuilder(SpatialReferences.getWgs84());
+    polylineBuilder.addPoint(new Point(-30, 20-offset));
+    polylineBuilder.addPoint(new Point(30, 20-offset));
```

Comment:
let's add some spaces here just for readability

```suggestion
    polylineBuilder.addPoint(new Point(-30, 20 - offset));
    polylineBuilder.addPoint(new Point(30, 20 - offset));
```